src/code_80850_len_3060.c

```diff
@@ -268,7 +268,121 @@ void enforce_hpfp_limits(void) {
     }
 }
 
-INCLUDE_ASM(s32, "code_80850_len_3060", initialize_status_menu);
+void initialize_status_menu() {
```

Comment:
void in the arguments for this func

Reply:
ahh yee, nice catch

if you weren't aware Rain, `()` vs `(void)` are actually kind of different. the latter specifically means 0 args, and the former means anything